batch/AWS_inference_runner.sh

```diff
@@ -66,29 +68,30 @@ fi
 
 python -m pip install --upgrade pip # needs new pip for toml file
 
-(cd COVIDScenarioPipeline && pip install -e gempyor_pkg)
+(cd $COVID_PATH && pip install -e gempyor_pkg)
```

Comment:
Does $COVID_PATH makes sense here as the tar.gz file should send the same folder

Reply:
(2 occurences)